packages/lib/services/spellChecker/SpellCheckerServiceDriverBase.ts

```diff
@@ -4,15 +4,15 @@ export default class SpellCheckerServiceDriverBase {
 		throw new Error('Not implemented');
 	}
 
-	public setLanguage(_v: string) {
+	public setLanguage(_v: any) {
 		throw new Error('Not implemented');
 	}
```

Comment:
It might make sense to rename `setLanguage(v: string)` to `setLanguages(_v: string[])`.

If this is the done, a short documentation comment that explains that single-language-supporting spell checkers should use the first language in `v_` would make sense here.

Reply:
I think this function name should be left as is. Since it base driver, we want to provide template for making other spell check drivers. For Electron it may be plural, but others may be not. Let abstract from quantity of the languages, and use single form, and let argument type dictate it. For Electron spell checker it's `string[]`.

Reply:
I agree with @personalizedrefrigerator actually. An array of strings is a superset of a single string, so if a driver only supports one language it will still work. It also means that we can have proper type checking at this level.

And finally that driver stuff is not really useful (my fault) since we'll probably only ever have one driver.

Reply:
> And finally that driver stuff is not really useful (my fault) since we'll probably only ever have one driver.

Could I then remove driver implementation and make it simpler? It will solve the problems with renaming `setLanguage` and I then can just remove `_language` from `addWordToSpellCheckerDictionary`.

Reply:
There's something I'm not getting with this setLanguage function. Why is it not possible to just rename it to setLanguages and make it accept an array of strings? Just keep the same arguments in the same order otherwise. I prefer to keep refactoring to a minimum. If we ever remove this driver abstraction that would be in a separate PR.

Reply:
Renamed function.